nixos/modules/hardware/video/nvidia.nix

```diff
@@ -594,15 +594,29 @@ in
               (lib.mkIf cfg.powerManagement.enable {
                 nvidia-suspend = nvidiaService "suspend";
                 nvidia-hibernate = nvidiaService "hibernate";
+                nvidia-hybrid-sleep = (nvidiaService "hibernate") // {
```

Comment:
Can you justify this service? It is not present in the upstream driver 

https://download.nvidia.com/XFree86/Linux-x86_64/550.142/README/powermanagement.html